client/jetpack-cloud/sections/partner-portal/assign-license-form/index.tsx

```diff
@@ -9,6 +9,7 @@ import SearchCard from 'calypso/components/search-card';
 import { SITE_CARDS_PER_PAGE } from 'calypso/jetpack-cloud/sections/partner-portal/assign-license-form/constants';
 import { useAssignMultipleLicenses } from 'calypso/jetpack-cloud/sections/partner-portal/hooks';
 import { addQueryArgs } from 'calypso/lib/url';
+import { isMultisiteCompatibleLicenseKeys } from '../utils';
```

Comment:
```suggestion
import { areLicenseKeysMultisiteCompatible } from '../utils';
```

Reply:
Thought I had pushed the file but this is fixed. 👍